libopenage/engine.cpp

```diff
@@ -47,6 +47,8 @@ Engine *Engine::instance = nullptr;
 void Engine::create(util::Dir *data_dir, const char *windowtitle) {
 	// only create the singleton instance if it was not created before..
 	if (Engine::instance == nullptr) {
+		// initializa the default actions
+		input::actions::initialize_actions();
```

Comment:
The current idea is to make an ActionManager class included in the engine. But it has to be initialized before anything that requires the actions.